config.md

```diff
@@ -4,9 +4,17 @@ Aardvark-dns will read configuration files from a given directory.
 
 Inside this directory there should be at least one config file. The name of the file equals the network name.
 
+### First line
 The first line in the config must contain a comma separated list of listening ips for this network, usually the bridge ips.
 At least one ip must be given.
+**Note**: Optional second coloumn allows users to specifify DNS servers at network level (seperated by comman), all the containers
```

Comment:
An optional second column of comma delimited domain name servers can be used at the network level.  All containers on that network will inherit all the specified name servers instead of using the host's resolver.

Reply:
Done.